### ISSUE TITLE
"High Sierra plus" build does not work on High Sierra

### ISSUE THREAD

**Issue by winzig** [wagerr/wagerr#90]
The Monte Carlo 2.0.1 release for "high-sierra-plus" does not load on High Sierra:

<img width="423" alt="screen shot 2018-10-06 at 4 33 52 pm" src="https://user-images.githubusercontent.com/127062/46576707-a15f2280-c985-11e8-9ef7-bc0d7104f384.png">

**Comment by Bayalu** [comment]
I have encountered this problem too. Until they fix this bug you can open the wallet using terminal. 
Applicationa
wagerr core- show package contents
contents
mac os
wagerr qt- open with terminal

**Comment by gulfcoastdev** [comment]
> Wagerr-Qt.app/Contents/MacOS/Wagerr-Qt ; exit;
> Segmentation fault: 11
> logout
> 

Doesn't work for me.

**Comment by Bayalu** [comment]
You have to open Wagerr-Qt with terminal. 
Right click on Wagerr-Qt.app/Contents/MacOS/Wagerr-Qt 
Open with Terminal. Once it runs with Terminal you won't have to do it again and you'll be able to run it from Wagerr's shortcut on desktop. 
It does work.

**Comment by gulfcoastdev** [comment]
No. It does not. I just pasted you terminal log with segmentation fault: 11 error that I am getting when I do it.

**Comment by Bayalu** [comment]
Did you run QT with Terminal?

**Comment by gulfcoastdev** [comment]
yeah

**Comment by Bayalu** [comment]
Reinstall QT and make sure you install the QT version that corresponds to your specific OS version. First time run it with Terminal then it will work fine.

**Comment by gulfcoastdev** [comment]
I did but it still doesn't work. 
I use 
https://github.com/wagerr/wagerr/releases/download/v2.0.1/wagerr-2.0.1-osx.dmg
on
Mac High Sierra. 
Is it correct release?

**Comment by Bayalu** [comment]
There are issue with the version you are trying to download. 
Use his one wagerr-2.0.1-osx-*.dmg or wagerr-2.0.1-osx-mojave.dmg

**Comment by gulfcoastdev** [comment]
Hmm perhaps I am missing something but I can't find under list of releases....

**Comment by gulfcoastdev** [comment]
https://github.com/wagerr/wagerr/releases

**Comment by Bayalu** [comment]
Try to install Mojave release even if you have High Sierra. 
https://github.com/wagerr/wagerr/releases/download/v2.0.1/wagerr-2.0.1-osx-mojave.dmg
Run it with terminal on mac (show package contents). Let me know if it works for you.

**Comment by gulfcoastdev** [comment]
Mojave worked! Thanks for help.

**Comment by Bayalu** [comment]
Np

**Comment by McLumnpy** [comment]
I tried to install this version on High sierra with Norton anti virus installed and was unable to run it as NAV found a file contained a virus: OSX.Trojan.Gen it deleted a file in the following path:-
Applications/Wagerr-Qt.app/Contents/MacOS/Wagerr-Qt. I've tried back to 1.5 and all have the same issue. Think I'll leave my wgr on waves until this is resolved.

**Comment by WagerrTor** [comment]
@McLumnpy 
We will publish with each release virustotal link and if there are any issues with any of antivirus software versions, you will be able to see it before running. Please follow the guides of your antivirus software how to whitelist this application and send report requests (if available) to your antivirus company/provider.

@winzig we updated macos part and further builds will require min. macos version 10.8, built for darwin14. I will make screenshots from different OS with current build which I created from current master and if they all work I will close this issue as it is resolved, please feel free to open it if the issue is not resolved in next release/current master.
